crates/hir-expand/src/attrs.rs

```diff
@@ -177,7 +177,7 @@ pub struct Attr {
     pub id: AttrId,
     pub path: Interned<ModPath>,
     pub input: Option<Interned<AttrInput>>,
-    pub ctxt: SyntaxContextId,
+    pub span: Span,
```

Comment:
This will have an impact on memory usage unfortunately (estimating 10mb on `self`), but we can't really avoid that